src/visuals-api.d.ts

```diff
@@ -1029,6 +1029,14 @@ declare module powerbi {
     export interface FilterTypeDescriptor {
         selfFilter?: boolean;
     }
+
+    export const enum PendingChangesType {
+      "Filters",
+    }
+
+    export type PendingChanges = {
+      [key in PendingChangesType]?: boolean;
```

Comment:
I think, as we are working with Enum, `key` is a bad naming here. Someone could expect that a `key` of enum element is used, while in fact `value` is used here. 
For example, in current realization, `0` value is used, and not the "Filters", what would be considered as a `key` from a first glance.